sklearn/metrics/pairwise.py

```diff
@@ -1015,19 +1015,35 @@ def paired_euclidean_distances(X, Y):
 
 
 def paired_manhattan_distances(X, Y):
-    """Compute the L1 distances between the vectors in X and Y.
+    """Compute the paired L1 distances between X and Y.
+
+    Distances are calculated between (X[0], Y[0]), (X[1], Y[1]), ...,
+    (X[n_samples], Y[n_samples]).
 
     Read more in the :ref:`User Guide <metrics>`.
 
     Parameters
     ----------
     X : array-like of shape (n_samples, n_features)
+        An array where each row is a sample and each column is a feature.
```

Comment:
Here, array-likes (numpy arrays, but also list and sparse matrices) are accepted.

Hence, I would suggest changing this to:
```suggestion
        An array-like where each row is a sample and each column is a feature.
```

Reply:
Makes a lot of sense to me, happy to add :)